src/main/java/io/meeds/qa/ui/pages/page/factory/space/ManageSpacesPage.java

```diff
@@ -706,6 +714,14 @@ public void checkSpaceBookmarkStatusFromTopbarSpacePopover( boolean ShouldBeBook
       }
     }
 
+   public void checkSpaceBookmarkStatusFromFromThirdNavigationLevel( boolean ShouldBeBookmarked){
```

Comment:
duplicated word in function name , to be changed :
checkSpaceBookmarkStatusFromThirdNavigationLevel